recipe/meta.yaml

```diff
@@ -18,10 +18,11 @@ source:
   #  folder: cub_v1.8.0  # so the source code is in cupy/cub_v1.8.0
 
 build:
-  number: 0
+  number: 1
   skip: true  # [cuda_compiler_version in (undefined, "None")]
   script:
     #- export CUB_PATH=${PWD}/cub_v1.8.0  # TODO(leofang): remove this
+    - {{ PYTHON }} setup.py build -j "$(nproc)" -vv
```

Comment:
Would use `${CPU_COUNT}`. This is a special conda-build environment variable, which is configured to prevent oversubscription (important for multitenant nodes that CI providers commonly use).

Reply:
Thanks, John! It's good to know. I'll change it and rerun CI. However, this PR is meant for testing parallel build speed, and it turns out it's not much faster because the main bottlenecks are:
1. The multi-stage environment setups in conda-build
2. Building `cupy.cuda.thrust` (lots of C++ templates)

So I am not sure if this change should be merged...

Reply:
Oh....Another reason could be our CI has `CPU_COUNT=2`...Any chance to increase the limit?
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=188760&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=b2a8456a-fb11-5506-ca32-5ccd32538dc0&l=192

Reply:
I think it's fixed by the VM used in the CI: `Standard_DS2_v2` has 2 vCPUs:
https://docs.microsoft.com/en-us/azure/virtual-machines/dv2-dsv2-series#dsv2-series